source/frontend/src/components/maps/MapLeafletView.tsx

```diff
@@ -0,0 +1,250 @@
+import axios from 'axios';
```

Comment:
ok, so looks like you are trying to split map specific functionality into this file -> map vs sidebar and layout functionality to mapview, and then cleaning up the container to not contain logic better suited to being in these two files.

Reply:
Thats correct. The map page (homepage) should as a container for the layout with the different components. The map itself is just a wrapper for the leaflet with the buttons.